bridges/pallet-ibc/src/port.rs

```diff
@@ -0,0 +1,43 @@
+use super::*;
+
+use crate::routing::Context;
+use ibc::core::{
+	ics05_port::{
+		capabilities::{Capability, CapabilityName},
+		context::{CapabilityReader, PortReader},
+		error::Error as ICS05Error,
+	},
+	ics24_host::identifier::PortId,
+};
+
+// TODO: Revisit when port binding is implemented
```

Comment:
is this an ibc-rs problem?

Reply:
It's a bit dependent on how we implement the router, reason why I left it out initially.
I think I can do the capability key storage now.